pkgs/applications/emulators/xemu/default.nix

```diff
@@ -89,20 +89,19 @@ stdenv.mkDerivation (self: {
     })
   ];
 
-  preConfigure = let
-    # When the data below can't be obtained through git, the build process tries
-    # to run `XEMU_COMMIT=$(cat XEMU_COMMIT)` (and similar)
-    branch = "master";
-    commit = "d8fa50e524c22f85ecb2e43108fd6a5501744351";
-    inherit (self) version;
-  in ''
+   preConfigure = let
+        # When the data below can't be obtained through git, the build process tries
+        # to run `XEMU_COMMIT=$(cat XEMU_COMMIT)` (and similar)
+        rev = "4a99fd0f1855fc52e15885812abd3ee4bb2b3847";
```

Comment:
How can we get `rev` hash from `src = fetchFromGitHub { rev = "v${version}" }` for this?

Reply:
Usually nix-prefetch scripts can do this. I need to seek an example code, tho.

Reply:
(Alternative). Don't bother putting in `master` and `rev`. A version alone is more than enough to fix the title.

Reply:
No answer. I put in alternative (omitting `master` and `rev`). New version bump too. Lmk.